### PR TITLE
Test against modern node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by alexanderdickson** [comment]
Do you know why the Node 6 build failed?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
-  - 0.10.26
\ No newline at end of file
+  - 6
+  - 4
+  - 0.10
```
Comment (stevemao) [inline .travis.yml]: `'0.12'` (quote)

**Comment by amilajack** [comment]
Fixed

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 language: node_js
 node_js:
-  - 0.10.26
\ No newline at end of file
+  - 6
+  - 4
+  - '0.12'
+  - '0.10'
```
Comment (stevemao) [inline .travis.yml]: `0.10` support is dropped :)

**Comment by stevemao** [comment]
tests failed because `grunt` is not a dep

**Comment by amilajack** [comment]
Made the changes :)

File: .travis.yml [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
 language: node_js
+
 node_js:
-  - 0.10.26
\ No newline at end of file
+  - 6
+  - 4
+  - '0.12'
+
+install:
```
Comment (stevemao) [inline .travis.yml]: this is not right... you need to add `grunt` in package.json

Comment (amilajack) [inline .travis.yml]: Should I add gulp or gulp-cli?

Comment (stevemao) [inline .travis.yml]: `grunt`

File: .travis.yml [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
```
Comment (stevemao) [inline .travis.yml]: `install:` could be removed

File: package.json [hunk 5]
```diff
@@ -2,30 +2,30 @@
...
-    "email": "alex@alexanderdickson.com",
-    "url": "http://alexanderdickson.com"
-  },
   "scripts": {
     "test": "grunt travis --verbose"
   },
   "devDependencies": {
+    "grunt": "^1.0.1",
     "grunt-cli": "~0.1.6",
+    "grunt-contrib-concat": "~0.3.0",
```
Comment (stevemao) [inline package.json]: I don't think this version is compatible with grunt `1.0.1`

Comment (amilajack) [inline package.json]: All test are passing. Seems like it worked

Comment (stevemao) [inline package.json]: > All test are passing. Seems like it worked

Not really :)

Comment (amilajack) [inline package.json]: https://travis-ci.org/alexanderdickson/waitForImages/builds/162658446

**Comment by amilajack** [comment]
@alexanderdickson Ready to merge :party

**Pull request merged**